api/overview/azure/iot.md

```diff
@@ -17,40 +17,75 @@ ms.service: multiple
 
 ## Overview
 
---Overview text here--
+[Azure IoT Hub](https://azure.microsoft.com/services/iot-hub/) is a fully managed service that enables reliable and secure bi-directional communications between millions of devices and a solution back end.
 
+Devices and data sources in an IoT solution can range from a simple network-connected sensor to a powerful, standalone computing device. Devices may have limited processing capability, memory, communication bandwidth, and communication protocol support. The IoT [device SDKs](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-sdks) enable you to implement client applications for a wide variety of devices and back-end applications.
 
-## Management library
+The device SDK for .NET facilitates building devices running .NET that connect to Azure IoT Hub.
 
---IoT management blurb here--
+The service SDK for .NET facilitates building back-end applications using .NET that manage and allow controlling devices from the Cloud.
 
-Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Management.IotHub) directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
+[Learn more about Azure IoT](https://docs.microsoft.com/azure/iot-hub/).
+
+
+## Client library
+
+Use the .NET IoT devices client to connect and send messages to your IoT Hub.
+
+Install the [NuGet package]( https://www.nuget.org/packages/Microsoft.Azure.Devices.Client) directly from the Visual Studio [Package Manager console](https://docs.microsoft.com/nuget/tools/package-manager-console) or with the [.NET Core CLI](https://docs.microsoft.com/dotnet/core/tools/dotnet-add-package).
 
 #### Visual Studio Package Manager
 
 ```powershell
-Install-Package Microsoft.Azure.Management.IotHub
+Install-Package Microsoft.Azure.Devices.Client
 ```
 
 ```bash
-dotnet add package Microsoft.Azure.Management.IotHub
+dotnet add package Microsoft.Azure.Devices.Client
 ```
+### Examples 
 
-### Example
-
---Example overview--
+This example connects to the IoT Hub and sends a message every 1 second.
 
 ```csharp
-/* Code goes here */
+int _messageId = 1;

```

Comment:
You can probably remove these variable declarations and just use them explicitly in the samples to reduce the surface area